paddlenlp/seq2vec/encoder.py

```diff
@@ -24,46 +24,51 @@
 
 
 class BoWEncoder(nn.Layer):
-    """
+    r"""
     A `BoWEncoder` takes as input a sequence of vectors and returns a
     single vector, which simply sums the embeddings of a sequence across the time dimension. 
     The input to this module is of shape `(batch_size, num_tokens, emb_dim)`, 
     and the output is of shape `(batch_size, emb_dim)`.
 
     Args:
-        emb_dim(int): It is the input dimension to the encoder.
```

Comment:
It is the input dimension to the encoder. 
->
The dimension of each vector in the input sequence.

Reply:
done